UnityProject/Assets/Scripts/Player/GhostMove.cs

```diff
@@ -67,7 +67,7 @@ public Vector3 MoveTowards(Vector3 current, Vector3 target, float maxDistanceDel
 	public void RPCUpdatePosition(Vector3 newPosition, int matrixID, OrientationEnum direction, bool @override,
 		bool Smooth)
 	{
-		if (isLocalPlayer && @override == false || isServer) return;
+		if (hasAuthority && @override == false || isServer) return;
```

Comment:
a lot of places changed from ``isLocalPlayer`` to ``hasAuthority``. Why is that?

Reply:
is local player is no longer a good Indicator of if this is the player controlling the object
Use has authority instead